client/pages/review-form.jsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+
+export default class ReviewForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      informButt: 'inform m-2 ms-0',
+      perButt: 'persuade m-2',
+      currentRating: null,
+      currentReview: ''
+    };
+    this.handleClickInform = this.handleClickInform.bind(this);
+    this.handleClickPersuade = this.handleClickPersuade.bind(this);
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleClickInform(event) {
+    this.setState({ informButt: 'inform-on m-2 ms-0', perButt: 'persuade-off m-2', currentRating: 'inform' });
+  }
+
+  handleClickPersuade(event) {
+    this.setState({ informButt: 'inform-off m-2 ms-0', perButt: 'persuade-on m-2', currentRating: 'persuade' });
+  }
+
+  handleChange(event) {
+    this.setState({ currentReview: event.target.value });
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+    const req = {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify({ articleInfo: this.props.info })
+    };
+    fetch('/api/article-review', req)
+      .then(res => res.json())
+      .then(result => {
+        const newArticleId = parseInt(result.articleId);
+        const req = {
+          method: 'POST',
+          headers: {
+            'Content-Type': 'application/json'
+          },
+          body: JSON.stringify(this.state)
+        };
+        fetch(`/api/user-review/${newArticleId}`, req)
+          .then(res => res.json())
+          .then(result => {
+          });
+      });
+    window.location.hash = '#user-reviews';
+  }
```

Comment:
You should move this hash update in between lines 52 and 53 so that the page doesn't switch until the POSTS are done